CONTRIBUTING.md

```diff
@@ -139,12 +139,12 @@ Here are the main files and folders of the project.
 
 ```
 ▸ src/
-  ▸ components/       << The pReact components for the UI of our widgets (UI)
-  ▸ connectors/       << The source of all the connectors (UX without UI)
-  ▸ css/              << The source of the themes (reset / algolia theme)
-  ▸ lib/              << The core of the library, instantsearch, url
-  ▸ widgets/          << The source of the widgets (UX + UI)
+  ▸ components/       << The Preact components for the UI of our widgets
+  ▸ connectors/       << The source of all the connectors driving the widgets' logic
   ▸ helpers/          << The source of the method helpers
+  ▸ lib/              << The core of the library (InstantSearch, routers, etc.)
+  ▸ types/            << The types describing the interfaces
```

Comment:
```suggestion
  ▸ types/            << The TypeScript declarations for global concepts
```

not sure if this is really better

Reply:
I just went for "The TypeScript declarations" for now.